pkgs/tools/misc/ntfy-sh/default.nix

```
@@ -15,8 +21,19 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
   preBuild = ''
-    make cli-deps-static-sites
+    sed -i 's# /bin/echo# echo#' Makefile
+    ln -s ${nodeDependencies}/lib/node_modules web/node_modules
+    DISABLE_ESLINT_PLUGIN=true make web-build docs-build
```

Looking at the repo, I don't see a reason for building the documentation -- it doesn't look like they get bundled into the binary in any way or anything like that.

Is there something I'm missing here?

```suggestion
    DISABLE_ESLINT_PLUGIN=true make web-build
```

---

They actually are embedded in the binary. [See here](https://github.com/binwiederhier/ntfy/blob/main/server/server.go#L86) and [here](https://github.com/binwiederhier/ntfy/blob/main/server/server.go#L402).
If we don't embed the docs, we'd have to patch something, because the build script errors while building:
```
Building subPackage .
server/server.go:86:13: pattern docs: no matching files found
error: builder for '/nix/store/mn2sq5f3b00j7qg9fd806vnal4rcwkgz-ntfy-sh-1.28.0.drv' failed with exit code 1;
```

---

Ah, got it, thanks! Sorry, I didn't actually test my change, and only grepped around with the GitHub search (which isn't the best, no clue why I didn't use CS).